src/main/java/com/nylas/Account.java

```diff
@@ -31,10 +34,14 @@ public Boolean getTrial() {
 		return trial;
 	}
 
+	public Map<String, String> getMetadata() {
+		return metadata;
+	}
+
 	@Override
 	public String toString() {
 		return "Account [id=" + getId() + ", billing_state=" + billing_state + ", email=" + email + ", provider=" + provider
-				+ ", sync_state=" + sync_state + ", trial=" + trial + "]";
+				+ ", sync_state=" + sync_state + ", trial=" + trial + ", metadata=" + metadata + "]";
```

Comment:
Can you use some automatic toString library such as https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/builder/ReflectionToStringBuilder.html?

Reply:
This is a really good shout, thanks @quzhi1! I don't think I'll tackle this during this PR, but I've noted it down because maintaining the `toString()` methods manually is a pain to say the least.